src/utils.ts

```diff
@@ -363,6 +363,17 @@ export const nFormatter = (num: number, digits: number): string => {
   );
 };
 
+export const formatSpeedBits = (speed: number): string => {
+  // source: https://en.wikipedia.org/wiki/Data-rate_units#Conversion_table
+  const suffix = ["bit/s", "kbit/s", "Mbit/s", "Gbit/s"];
```

Comment:
```suggestion
  const suffix = ["bps", "kbps", "Mbps", "Gbps"];
```
We can use the standard abbreviations to keep it short.